scripts/release.sh

```diff
@@ -3,34 +3,50 @@
 set -e
 
 source ${DAPPER_SOURCE}/scripts/lib/utils
+source ${SCRIPTS_DIR}/lib/debug_functions
+
+### Functions ###
+
+function determine_repo() {
+    origin_url=$(git config --get remote.origin.url)
+    if [[ "$origin_url" =~ ^https:.* ]]; then
+        echo "$origin_url" | cut -f 4 -d'/'
+    elif [[ "$origin_url" =~ ^git@.* ]]; then
+        echo "$origin_url" | cut -f 4- -d'/' | cut -f 1 -d '.'
```

Comment:
This doesn’t work for me. I think a better solution, which would handle both `https:` and `git@`, would be to use

    basename "$origin_url" .git

(note that `https:` URL can _also_ end with `.git`).

Reply:
The point is to extract e.g. `submariner-io` from a URL such as `https://github.com/submariner-io/submariner` which is what CI uses.

What I get from your proposal is `submariner`.

I suppose we could use `basename $(dirname https://github.com/submariner-io/submariner)` ?

FYI, this script won't work locally anyhow unless you set a bunch of secrets to access your repos and Quay. I'm not sure it's worthwhile to even make it work locally.

Reply:
Oops, sorry, I thought we were looking for the repo, not the group.

Reply:
Probably the naming is confusing, I'll change it to `determine_org`